docs/oid-info.md

```diff
@@ -2,52 +2,89 @@
 
 ## Description
 
-Sigstore maintains its own Private Enterprise Number (57264) with the Internet
+Sigstore maintains its own Private Enterprise Number ([57264](http://oid-info.com/get/1.3.6.1.4.1.57264)) with the Internet
 Assigned Numbers Authority to help identify and organize additional metadata in
 code signing certificates issued by Fulcio instances. This document aims to
 provide a simple directory of values in use with an explanation of their
 meaning.
 
+## Requirements to support signing with CI/CD workload identities
+
+In order to support Sigstore code signing with CI/CD based workfload identities the following claims must be included in the OIDC ID Token. See example claim values for each extension in the detailed [Directory](#directory). 
+
+Requirements:
+
+- MUST include `iss` claim for `Issuer` extension.
+- MUST include claim to support: `Build Signer URI` that identifies the specific build instructions that are responsible for signing.
+- MUST include claim to support: `Build Signer Digest` which is an immutable reference to a specific version of the build instructions that are responsible for signing.
+- MUST include claim to support: `Runner Environment` that differentiates between builds that took place in platform-hosted cloud infrastructure or customer-hosted infrastructure.
+
+Recommended:
+
+- SHOULD include `iss` that uniquely identifies ID tokens originating from the CI/CD system, e.g. not shared with OIDC OAuth 2.0 tokens for email/username logins.
```

Comment:
What is the reason for this requirement?

Reply:
Makes sense, although you could still run into issues if the OIDC issuers are run within the same trust domain such that if one is compromised, both could be. For example a CI issuer issuing a token for an email address, or an email-based issuer issuing a token for a CI identity (we do add some checks in fulcio to make sure a subject looks like an email when it's expected, I would hesitate to always enforce that a CI identity is not an email though)